guides/common/modules/con_working-with-host-groups.adoc

```diff
@@ -0,0 +1,6 @@
+[id="Working_with_Host_Groups_{context}"]
+= Working with Host Groups
+
+A host group is a template for common host settings.
+Adding host settings and attributes for every new host is time consuming.
+If you use host groups, you can apply common attributes to hosts that you create.
```

Comment:
Perhaps this intro could be rephrased to make a bit clearer that it's "define settings once, apply them many times"?
I'm not sure using the term "template" is a good idea, because it's a technical term for stuff like job templates, provisioning templates, report templates etc. (Non-blocker, I don't have a better idea.)
Also, inconsistency in terminology -- attributes vs. properties. We should stick with a single term, ideally.

Reply:
Re: 'templates' - We're consistently inconsistent. We do similar things all over the place: content sources vs. Alternate Content Sources; host collections vs. host groups; permissions filters vs. content view filters; etc.

Maybe "acts like a template" is a bit clearer? But I think using "template" is fine, given the above.

Reply:
I rewrote the intro in the spirit you suggested @Lennonka, let me know if it doesn't work for you like this.

I kept the word "template" here because it's actually a more accurate description than "[host] group". See also https://bugzilla.redhat.com/show_bug.cgi?id=2253496#c12 (private comment explaining that "template" describes the behavior of host settings inheritance better than "group"). I understand that the word is used for different purposes within the project, but I believe it's still worth it to use it here as well.